dali/ft/filecopy.cpp

```diff
@@ -592,6 +596,24 @@ FileSprayer::FileSprayer(IPropertyTree * _options, IPropertyTree * _progress, IR
     encryptKey.set(options->queryProp(ANencryptKey));
     decryptKey.set(options->queryProp(ANdecryptKey));
 
+    fileUmask = -1;
+    if (options->hasProp(ANumask))
+    {
+        StringBuffer umaskStr;
+        options->getProp(ANumask, umaskStr);
```

Comment:
A umask of 0 is wide open and should not be used unless it was really specified, so I want to be sure not to return 0 if a bad value was in the env.